src/packages/crypto/crypto.cc

```diff
@@ -102,7 +102,7 @@ void f_hash() {
 
 result:
   if (!result) {
-    error("hash() unknown hash type: %s", algo);
+    error("hash() unknown hash type: %s\n", algo);
```

Comment:
The error msg should have an trailing newline

Reply:
I added one because it was missing. Does it need removed?